examples/platform/linux/AppMain.cpp

```diff
@@ -491,6 +497,16 @@ int ChipLinuxAppInit(int argc, char * const argv[], OptionSet * customOptions,
     err = GetPayloadContents(LinuxDeviceOptions::GetInstance().payload, rendezvousFlags);
     SuccessOrExit(err);
 
+    // We need to set DeviceInfoProvider before Server::Init to setup the storage of DeviceInfoProvider properly.
```

Comment:
```suggestion
    // We need to set DeviceInfoProvider before Server::Init to set up the storage of DeviceInfoProvider properly.
```
I know you just copied this...